roles/ansible_tower/tasks/main.yaml

```diff
@@ -30,3 +30,11 @@
 
 - name: Set (httpd_can_network_connect_db) flag on and keep it persistent across reboots
   seboolean: name=httpd_can_network_connect_db state=yes persistent=yes
+
+- name: Setup proot to allow access to /etc/tower/
+  lineinfile:
+    dest: /etc/tower/settings.py
+    backrefs: yes
+    regexp: '^(AWX_PROOT_SHOW_PATHS)( *)='
+    line: '\1\2= ["/etc/tower/"]'
```

Comment:
Since this is public, are we sure everyone wants this option changed?

What @mwoodson and I discussed last week was passing in an array of options we want set, and then doing a lineinfile for each.

Similar to what we do here (except we'll pass in the array that with_items will iterate over):
https://github.com/openshift/openshift-ansible/blob/1f9c84c2ac6ecb044b1ec5be4c21d864b20e996d/roles/openshift_sdn_node/tasks/main.yml#L21

That way only settings that someone wants will get set. We won't globally force people to use the options we want.